### PR TITLE
Nova chance: featured projects, what is glitch, new stuff are now in react

### PR DESCRIPTION
**Pull request by GregWeil**
- using `<details>` for the what is glitch video (no fallback yet)
- added an `outer` prop to `PopoverContainer` which is rendered before children, using the same args. This way the header can get the visibility callback without being in the onclickoutside dom
- removed all of the global click logic because now all popups and overlays are react :sparkles:
- get/set user prefs are only used by the new stuff popup, so it could probably start living in react. I'm leaving it for now because we have some nice localstorage checks on it that are not shared with currentUser but should be

### CODE REVIEW

File: src/presenters/pop-overs/popover-container.jsx [hunk 1]
```diff
@@ -13,26 +13,27 @@ popover pans, which have straight-walled sides rather than angled.
...
     // We need to set up and instantiate an onClickOutside wrapper
     // It's important to instantiate it once and pass though its children,
-    // otherwise the diff algorithm won't be able to figureo out our hijinks.
+    // otherwise the diff algorithm won't be able to figure out our hijinks.
     // https://github.com/Pomax/react-onclickoutside
 
     // We do extra work with disableOnClickOutside and handleClickOutside
     // to prevent event bindings from being created until the popover is opened.
     const clickOutsideConfig = {
-      handleClickOutside: (function() { return this.handleClickOutside; }).bind(this),
```
Comment (judeallred) [inline src/presenters/pop-overs/popover-container.jsx]: nova-chance.glitch.me/ doesn't currently work in Edge, it fails due to javascript exceptions that are rooted in this file.  I _think_ it has to do with this change here, which has to do with Edge handling function versus lambda binding differently than the others.

here's the full trace i see in the edge console:
```
The above error occurred in the <OnClickOutside(Wrapper)> component:
    in OnClickOutside(Wrapper) (created by PopoverContainer)
    in PopoverContainer (created by NewProjectPopContainer)
    in NewProjectPopContainer (created by Header)
    in nav (created by Header)
    in header (created by Header)
    in Header (created by PopoverContainer)
    in PopoverContainer (created by NewStuff)
    in NewStuff (created by UserPref)
    in UserPref (created by UserPref)
    in UserPref (created by NewStuffContainer)
    in NewStuffContainer (created by HeaderContainer)
    in HeaderContainer

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
react-bundle.js (14236,5)

 The above error occurred in the <OnClickOutside(Wrapper)> component:
    in OnClickOutside(Wrapper) (created by PopoverContainer)
    in PopoverContainer (created by SignInPopContainer)
    in SignInPopContainer (created by Header)
    in nav (created by Header)
    in header (created by Header)
    in Header (created by PopoverContainer)
    in PopoverContainer (created by NewStuff)
    in NewStuff (created by UserPref)
    in UserPref (created by UserPref)
    in UserPref (created by NewStuffContainer)
    in NewStuffContainer (created by HeaderContainer)
    in HeaderContainer

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
react-bundle.js (14236,5)

 The above error occurred in the <OnClickOutside(Wrapper)> component:
    in OnClickOutside(Wrapper) (created by PopoverContainer)
    in PopoverContainer (created by NewStuff)
    in NewStuff (created by UserPref)
    in UserPref (created by UserPref)
    in UserPref (created by NewStuffContainer)
    in NewStuffContainer (created by HeaderContainer)
    in HeaderContainer

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
react-bundle.js (14236,5)

 Warning: Can't call setState (or forceUpdate) on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in NewProjectPopContainer (created by Header)
    in nav (created by Header)
    in header (created by Header)
    in Header (created by PopoverContainer)
    in PopoverContainer (created by NewStuff)
    in NewStuff (created by UserPref)
    in UserPref (created by UserPref)
    in UserPref (created by NewStuffContainer)
    in NewStuffContainer (created by HeaderContainer)
    in HeaderContainer
dependencies.js (4379,7)

4 SCRIPT5007: Unable to get property 'handleClickOutside' of undefined or null reference
```

**Comment by GregWeil** [comment]
thanks for testing! i tried a new approach that seems like it should be pretty safe. if it still doesn't work then I'll just revert it back to how it was

also i fixed a bug with asset resizing, we were treating everything as a large cover

**Comment by GregWeil** [comment]
small bonus:
- hopefully fix https://sentry.io/fog-creek-software/community/issues/613797036/
- updated some npm packages too

**Comment by judeallred** [comment]
bug looks fixed;  code lgtm!

In Edge, when i click on 'How it works in 1 minute', nothing happens;  i presume this due to the lack of <details> support

File: webpack.config.js [hunk 2]
```diff
@@ -56,6 +57,9 @@ module.exports = () => {
           },
         },
       },
+      minimizer: [
+        new UglifyJsPlugin({uglifyOptions: {safari10: true}}),
```
Comment (judeallred) [inline webpack.config.js]: cool!

Comment (GregWeil) [reply webpack.config.js]: but why isn't that enabled by default? (i have no idea what that flag does, just that it fixes an issue with `for (let ...)` in safari 10)

**Comment by judeallred** [comment]
polyfill works for me in edge 👍

**Pull request merged**